examples-staging/assets-cloud/schema.graphql

```diff
@@ -426,6 +427,11 @@ enum KeystoneAdminUIFieldMetaItemViewFieldMode {
   hidden
 }
 
+enum QueryMode {
+  default
+  insensitive
+}
```

Comment:
Note the naming and enum values here align with the `mode` field that will be in the new filters for strings (and since the new filters align with Prisma, so does this).